WallpaperChanger/__main__.py

```diff
@@ -11,7 +11,13 @@ def main():
     initialise_logging(args)
     logging.info("")
     logging.info("App started!")
-    app.App(args)
+
+    try:
+        app.App(args)
+    except KeyboardInterrupt:
+        logging.info("Keyboard Interrupt! Exiting app...")
+
```

Comment:
Maybe add any unknown exceptions here. Would be good to see so we can patch things up.